aws/cloudformation/standalone/marketing/deployment/marketing.yml.erb

```diff
@@ -507,6 +514,24 @@ Resources:
     UpdateReplacePolicy: Delete
     DeletionPolicy: Delete
 
+  MarketingStaticAssetsBucket:
+    Type: AWS::S3::Bucket
+    Properties:
+      BucketName: !Sub "cdo-marketing-static-assets-${Environment}"
```

Comment:
We’ve run into issues with our monolith application where `Environment` actually means `Environment Type` and because there are multiple deployments (Stacks) that use the same Environment Type, this is no longer a unique identifier.

I’d prefer to switch to using the [naming conventions that we adopted with Javabuilder](https://github.com/code-dot-org/javabuilder/blob/0ad7f3de06201e90685359ba45ffb0c537d2f1f9/cicd/3-app/javabuilder/template.yml.erb#L476).

Reply:
I've updated this to reflect this. Currently it will result in `cdo-dev-dev.marketing-static-assets`. However, once we start creating a new stack we can remove the extra `dev.marketing` part to just `marketing`.